gdx/src/com/badlogic/gdx/utils/ScreenCapturer.java

```diff
@@ -0,0 +1,185 @@
+/*******************************************************************************
+ * Copyright 2011 See AUTHORS file.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+package com.badlogic.gdx.utils;
+
+import java.nio.ByteBuffer;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.files.FileHandle;
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.Pixmap;
+import com.badlogic.gdx.graphics.PixmapIO;
+
+public class ScreenCapturer {
+	 private static int screenshotNumber = 1;
+	 private static boolean grayScale = false;
+	 private static boolean effects = true;
+	 
+	 
+	 /**
+	  * Saves a screenshot of a portion of a screen in the .png format 
+	  * @param x Starting xCoord
+	  * @param y Starting yCoord
+	  * @param width Width of the square zoom
+	  * @param height Height of the square zoom
+	  */
+	 public static void saveScreenshotZoom(int x, int y, int width, int height){
```

Comment:
This is a weird name for this method, since you are not actually zooming anything. I'd just call this saveScreenshot. The docs also indicate that the the clip rectangle should be a square, but there is no such constraint.